### DISCUSSION TITLE
Developer to stable

### DISCUSSION THREAD

**Discussion by GarethHumbel** [YetiForceCompany/YetiForceCRM#16143] [General] [answered]
Does developer version upgrading to next related stable, or upgrade must be processed on stable version?

**Comment by rskrzypczak** [comment, marked as answer]
Update packages are only created between stable versions.
There is a high probability that the update from the intermediate version, i.e. the development version, will work, but nobody here will give you a guarantee.
FYI: https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/system-update
